content/blog/2022-06-07-using-gcp-remotes-in-dvc.md

```diff
@@ -0,0 +1,178 @@
+versions of the same datasets to train a model, so it’s crucial to have a tool
+that enables you to do this quickly and easily. That’s why we’re going to do a
+quick walkthrough of how to set up a remote in a GCP storage bucket and handle
+data versioning with [DVC](https://dvc.org/doc).
+
+We’ll start by creating a new storage bucket in our GCP account, then we’ll show
+how you can add DVC to your project, and finally, we’ll make updates to the
+dataset with DVC commands. We’ll be working with
+[this repo](https://github.com/iterative/stale-model-example) if you want an
+example to play with. By the time you finish, you should be able to create this
+setup for any machine learning project using an GCP remote.
+
+## Set up a GCP storage bucket
+
+Make sure that you already have a
+[GCP account](https://console.cloud.google.com). You’ll need a valid credit card
+to create a new account. Once you’re logged into your account, you should see a
+screen like this with some of the services GCP offers.
+
+![GCP initial page](/uploads/images/2022-06-07/gcp_initial_page.png)
+
+From here, you'll need to create a new project. Search for "create a project"
+and click the "IAM & Admin" option. You'll enter the name of the project, which
+is `Bicycle Project`, and choose the organization and location and click the
+`Create` button. This will take you to your project dashboard and show you all
+of the stats and settings you have available.
+
+![create a new GCP project](/uploads/images/2022-06-07/gcp_new_project.png)
+
+Then you need to go to `Cloud Storage` in the left sidebar to create a bucket to
+store the data. When you get to the Cloud Storage page, you should see something
+similar to this and you’ll click the `Create Bucket` button.
+
+![create_gcp_bucket.png](/uploads/images/2022-06-07/create_gcp_bucket.png)
+
+The Bucket page will have a lot of configurations you can set, but you can leave
+the settings in the default state if there’s nothing you need to customize. We
+have named this example bucket `updatedbikedata` as you can see below.
+
+![gcp_bucket_options.png](/uploads/images/2022-06-07/gcp_bucket_options.png)
+
+Now you can save your changes and you’ll be redirected to the `Bucket Details`
+page and you’ll see the bucket you just created.
+
+![created_gcp_bucket.png](/uploads/images/2022-06-07/created_gcp_bucket.png)
+
+### Get your credentials
+
+Since you have the bucket created, we need to get the credentials to connect the
+GCP remote to the project. Go to the `IAM & Admin` service and go to
+`Service Accounts` in the left sidebar.
+
+![no service accounts](/uploads/images/2022-06-07/gcp_empty_service_account.png)
+
+Click the `Create Service Account` button to create a new service account that
+you'll use to connect to the DVC project in a bit. Now you can add the name and
+ID for this service account and keep all the default settings. We've chosen
+`bicycle-service-account` for the name and `bicycle-account` for the ID. Click
```

Comment:
Any reason to use different account name and ID? Not a big deal, but it defaults to the same name for both, so this seemed like an unusual extra step when following along.

Reply:
That's another weird thing! It didn't default for me which is why they ended up different.